base_rest/components/service.py

```diff
@@ -151,8 +150,7 @@ def _secure_output(self, method, response):
         v = Validator(schema, purge_unknown=True)
         if v.validate(response):
             return v.document
-        _logger.error("Invalid response %s", v.errors)
-        raise UserError(_('Invalid Response'))
+        raise UserError(_('Invalid Response %s') % v.errors)
```

Comment:
IMO output validation error is a system error, not a user error

Reply:
because it's the developer's fault, not the user's

Reply:
it makes sense. I'll change